app/models/user.rb

```diff
@@ -26,6 +30,8 @@ class User < ApplicationRecord
   validates :password, presence: true, length: {minimum: Settings.password_minimum}
   
   scope :search_name, ->name{where "name LIKE ?", "%#{name}%"}
+
+  delegate :name, :email, to: :sender_recipients, prefix: :user
```

Comment:
giả sử cái sender_recipients nil thì cái này có lỗi như nào

Reply:
Module::DelegationError: User#name delegated to sender_recipient.name, but sender_recipient is nil

Reply:
@binh168 vậy chú điều chỉnh để cho nó ko thể raise cái kia lên nữa nhé